start/manage/index.php

```diff
@@ -3,31 +3,51 @@
 
 use Service\Container;
 
+$breadcrumbItems = [];
+$searchName = null;
 $container = new Container($configuration);
 $shipLoader = $container->getShipLoader();
 $ships = $shipLoader->getShips();
 
+if ($_SERVER['REQUEST_METHOD'] == 'POST' && array_key_exists('searchName', $_POST)) {
+    $searchName = $_POST['searchName'];
+    $ships = $shipLoader->searchByName($searchName);
+}
+
 ?>
+<form action='/manage/index.php' method='POST' class='form-inline'>
+    <div class='form-group'>
+        <label for='searchName'>Search Ships: </label>
+        <input id='searchName' name='searchName' class="form-control" value='<?php if ($searchName !== null) { echo $searchName;}?>'/>
+        <button type="submit" class="btn btn-success">Search</button>
+        <a href="/manage/index.php" class="btn btn-primary">Clear</a>
```

Comment:
When you put a link like this inside a form the default behavior is to submit the form. Which in this case, works. But I think you should add `type='button'` onto this so it knows its a link, not a submit button.